api/enki.py

```diff
@@ -494,26 +528,28 @@ def patron_activity(self, patron, pin):
                 raise CirculationException(response.content)
         for loan in result["checkedOutItems"]:
             yield self.parse_patron_loans(loan)
-        for type, holds in list(result["holds"].items()):
-            for hold in holds:
-                yield self.parse_patron_holds(hold)
```

Comment:
Do we not expect hold information from enki anymore? I can see the parsing methods were never implemented.